index.js

```diff
@@ -1,46 +1,69 @@
+require("./messages");
+var steam = require("steam");
+
+const DOTA_APP_ID = 570;
+
 var EventEmitter = require('events').EventEmitter,
     fs = require("fs"),
+    path = require("path"),
     util = require("util"),
     bignumber = require("bignumber.js"),
-    Schema = require('protobuf').Schema,
-    base_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/base_gcmessages.desc")),
-    gcsdk_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/gcsdk_gcmessages.desc")),
-    dota_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/dota_gcmessages.desc")),
-    protoMask = 0x80000000,
+    ProtoBuf = require('protobufjs'),
     Dota2 = exports;
 
-var Dota2Client = function Dota2Client(steamClient, debug, debugMore) {
+ProtoBuf.convertFieldsToCamelCase = true;
+var builder = ProtoBuf.newBuilder();
+ProtoBuf.loadProtoFile(path.join(__dirname, './proto/base_gcmessages.proto'), builder);
+ProtoBuf.loadProtoFile(path.join(__dirname, './proto/gcsdk_gcmessages.proto'), builder);
+ProtoBuf.loadProtoFile(path.join(__dirname, './proto/dota_gcmessages_client_fantasy.proto'), builder);
+ProtoBuf.loadProtoFile(path.join(__dirname, './proto/dota_gcmessages_client.proto'), builder);
+ProtoBuf.loadProtoFile(path.join(__dirname, './proto/dota_gcmessages_common.proto'), builder);
+ProtoBuf.loadProtoFile(path.join(__dirname, './proto/steammessages.proto'), builder);
+Dota2.schema = builder.build();
+
+var Dota2Client = function Dota2Client(steamUser, debug, debugMore) {
```

Comment:
Yesss, this part bugged me the most in the whole migration processus, didn't know you could create the user from the client. I've changed the method signature back to the original and committed the changes.